components/RecipeName.js

```diff
@@ -7,13 +7,13 @@ const RecipeName = ({ recipe, setRecipe, missing }) => {
     return (
         <>
             <View style={styles.heading}>
-                <Text>Title</Text>
+                <Text>Recipe Name</Text>
```

Comment:
Are we changing it from Title to Recipe Name?

Reply:
Ryan's design says "Recipe Name."

Reply:
My only concern is conflicting naming across our app. If we change it in one spot, we should change it in every instance to maintain consistency.